### ISSUE TITLE
Example refers to hardcoded path

### ISSUE THREAD

**Issue by chillenzer** [edbennett/glue_analysis#4]
Example is not actually run-able as there is a hardcoded path and no way to get that data.

**Comment by edbennett** [comment]
Yep, was dumped in there so you had something to look at rather than being well-considered as a useful example for others. I can send you the data if you want to try running it.

**Comment by chillenzer** [comment]
Please, do so. Thanks!
